pkg/policies/flowcontrol/resources/fluxmeter/flux-meter.go

```diff
@@ -221,9 +221,9 @@ func (fluxMeter *FluxMeter) setup(lc fx.Lifecycle, prometheusRegistry *prometheu
 	})
 }
 
-// GetFlowSelector returns the selector.
-func (fluxMeter *FluxMeter) GetFlowSelector() *policylangv1.FlowSelector {
-	return fluxMeter.flowSelector
+// GetSelectors returns the selectors.
+func (fluxMeter *FluxMeter) GetSelectors() []*policylangv1.Selector {
+	return fluxMeter.fluxMeterProto.GetSelectors()
 }
```

Comment:
This is not required as FlowSelector has been deprecated and won't be processed anymore. It will be removed from the specification in the future versions.